frontend/javascripts/oxalis/geometries/skeleton.js

```diff
@@ -485,6 +489,36 @@ class Skeleton {
     });
   }
 
+  /**
+   * Updates a node's position and that of its edges in a WebGL buffer.
+   */
+  updateNodePosition(treeId: number, nodeId: number, position: Vector3, tree: Tree) {
```

Comment:
Isn't the `treeId` already contained in the `tree` object?

Reply:
Alternatively, I think this method could do the `const tree = skeletonTracing.trees[treeId];` part on its own, since the `tree` is not used on the call site (except for passing it).